nephio-ansible-install/README.md

```diff
@@ -99,9 +101,10 @@ ansible-galaxy collection install community.docker # required for gitea
 Now that the environment is up an running we can install the Nephio environment
 
 First we create some prerequisites, which installs kubectl, kind, kpt, cni and setup the bash environment
+-kK is for password promt to allow script to install dependencies.
```

Comment:
This change is to allow user to enter the sudo password while running the ansible playbook else gets an error "sudo password is missing"